lib/cucumber/glue/registry_and_more.rb

```diff
@@ -155,6 +155,14 @@ def create_expression(string_or_regexp)
         raise ArgumentError.new('Expression must be a String or Regexp')
       end
 
+      def self.cli_snippet_type_options
```

Comment:
Moved this above the private section as there are [places in the repo we call it directly](https://github.com/cucumber/cucumber-ruby/blob/master/lib/cucumber/cli/options.rb#L329)